conversion-measurement/functions/views/adtech/ad-click.pug

```diff
@@ -9,5 +9,5 @@ div Ad (
   code a
   |  element):
 br
-a(href=href src='/blue-shoes.png' alt='blue shoes' attributionsrc=attributionsrc target="_blank") Click me
+a(href=href alt='blue shoes' attributionsrc=attributionsrc target="_blank") Click me
```

Comment:
To clarify: We're intentionally using `attributionsrc` with a value here, instead of using valueless `attributionsrc`?

That's certainly valid, but I wonder if we should encourage the valueless version in order to eliminate the background request. That would require changing this so that the `href` response does the registration, however.

Reply:
Using a valueless attributionsrc SGTM! Updated, PTAL.